.pre-commit-config.yaml

```diff
@@ -24,7 +24,7 @@ repos:
           - einops>=0.6.0
           - kornia>=0.6.9
           - lightning>=2.0.9
-          - matplotlib>=3.8.1
+          - matplotlib>=3.8.1,!=3.9.1.*
```

Comment:
The trailing `.*` is needed because of a 3.9.1.post1 release and should skip both 3.9.1 and 3.9.1.post1 according to my interpretation of https://packaging.python.org/en/latest/specifications/version-specifiers/#version-exclusion. 3.9.1 (without a trailing `.*`) would apparently not skip 3.9.1.post1. Would be great if someone could test to confirm though.

Reply:
For the record, this did work 👍